recipes/Time_Series/Preprocessor_Use_and_Performance_Evaluation.ipynb

```diff
@@ -113,7 +112,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "DATA_FILE_PATH = \"~/Downloads/energy_dataset.csv.zip\""
+    "DATA_FILE_PATH = \"hf://datasets/vitaliy-sharandin/energy-consumption-hourly-spain/energy_dataset.csv\""
```

Comment:
How does this work? Does pandas.read_csv() recognize this protocol?

Reply:
Apparently, yes it does. The "hf://" syntax is directly from the HF dataset page when you choose "use this dataset".